### PR TITLE
Minor Changes in CF660F

### PR DESCRIPTION
**Pull request by dustin-miao**
_If any of the below don't apply to this Pull Request, mark the checkbox as done._

- [ ] I have tested my code.
- [ ] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [ ] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions).
  - I understand that if it is clear that I have not attempted to follow these conventions, my PR will be closed.
  - If changes are requested, I will re-request a review after addressing them.

### CODE REVIEW

**Comment by dustin-miao** [comment]
Addresses #3574. Also fixed some minor mistakes and reformatted code.

File: solutions/platinum/cf-660F.mdx [hunk 1]
```diff
@@ -68,40 +61,41 @@ $$
...
-	}
-};
+template<typename T>
+T floor_div(T a, T b) {
+	return a / b - ((a ^ b) < 0 && a % b);
+}
 
-struct LineContainer : multiset<Line> {
-	long long div(long long a, long long b) { return a / b - ((a ^ b) < 0 && a % b); }
+struct line_container_line {
```
Comment (SansPapyrus683) [inline solutions/platinum/cf-660F.mdx:77]: rename to LineContainerLine  & we're done

File: solutions/platinum/cf-660F.mdx [hunk 2]
```diff
@@ -133,7 +125,7 @@ int main() {
 		ips[i] = ips[i - 1] + A[i] * i;
 	}
 
-	LineContainer lc;
+	line_container lc;
```
Comment (SansPapyrus683) [inline solutions/platinum/cf-660F.mdx:128]: are you sure you renamed this properly

File: solutions/platinum/cf-660F.mdx [hunk 3]
```diff
@@ -68,40 +61,41 @@ $$
...
-			x->p = div(y->m - x->m, x->k - y->k);
+			x->p = floor_div(y->b - x->b, x->m - y->m);
 		return x->p >= y->p;
 	}
 
-	void add(long long k, long long m) {
-		auto z = insert({k, m, 0}), y = z++, x = y;
+public:
+	void add(long long m, long long b) {
+		auto z = insert({m, b, 0}), y = z++, x = y;
```
Comment (SansPapyrus683) [inline solutions/platinum/cf-660F.mdx:98]: put statements on separate lines

**Comment by dustin-miao** [comment]
Updated code compiles and works: https://codeforces.com/contest/660/submission/200493472

File: solutions/platinum/cf-660F.mdx [hunk 4]
```diff
@@ -68,40 +61,42 @@ $$
...
+class LineContainer : multiset<LineContainerLine, less<>> {
 	bool isect(iterator x, iterator y) {
 		if (y == end()) {
 			x->p = LLONG_MAX;
 			return false;
 		}
-		if (x->k == y->k)
-			x->p = x->m > y->m ? LLONG_MAX : -LLONG_MAX;
+		if (x->m == y->m)
+			x->p = x->b > y->b ? LLONG_MAX : LLONG_MIN;
```
Comment (SansPapyrus683) [inline solutions/platinum/cf-660F.mdx:89]: braces

**Review by SansPapyrus683 — APPROVED** [review]
yeah whatever

**Pull request merged**